tests/paramgrill.c

```diff
@@ -56,17 +54,57 @@ static const int g_maxNbVariations = 64;
 *  Macros
 **************************************/
 #define DISPLAY(...)  fprintf(stderr, __VA_ARGS__)
+#define TIMED 0
+#ifndef DEBUG
+#  define DEBUG 1
+#endif
+#define DEBUGOUTPUT(...) { if (DEBUG) DISPLAY(__VA_ARGS__); }
 
 #undef MIN
 #undef MAX
 #define MIN(a,b)   ( (a) < (b) ? (a) : (b) )
 #define MAX(a,b)   ( (a) > (b) ? (a) : (b) )
 #define CUSTOM_LEVEL 99
 
+/* indices for each of the variables */
+#define WLOG_IND 0
```

Comment:
Another possibility for such usage is to use `enum` instead.
This makes it possible to create a dedicated type, which tends to help later on, as the compiler can distinguish type errors.